src/graph/viewport.ts

```diff
@@ -5,12 +5,26 @@ import * as animate from '@f/animate';
 import { removeClass, forEachNode, stringToSvg } from '../utils/';
 import { typeNameToId } from '../introspection';
 
+interface Point {
```

Comment:
@mac2000 Looks good 👍 
Please add `FIXME` comment that references your PR in SvgPanAndZoom so latter we can remove this code.

Reply:
Yep, that is really good point, thank you, did add as much as possible declarative `FIXME` comment